src/sc/proofverifier.cpp

```diff
@@ -81,8 +87,15 @@ CCertProofVerifierInput CScProofVerifier::CertificateToVerifierItem(const CScCer
     certData.proof = certificate.scProof;
     certData.verificationKey = scFixedParams.wCertVk;
 
+    if (view != nullptr) {
+        if (scFixedParams.version >= 2) {
+            auto const& cdh = view->GetActiveCertView(certData.scId).certDataHash;
+            certData.lastCertHash = cdh.IsNull() ? CFieldElement::GetPhantomHash() : cdh;
+        }
+    }
```

Comment:
Save one indentation level if possible

```suggestion
    if (view && scFixedParams.version >= 2) {
        auto const& cdh = view->GetActiveCertView(certData.scId).certDataHash;
        certData.lastCertHash = cdh.IsNull() ? CFieldElement::GetPhantomHash() : cdh;
    }
```

Reply:
Ok, done in 7e39749d9b01b1ee9557e5bc221d95963163f968